Projects/Feature/Scenes/ArtistScene/ViewModel/ArtistViewModel.swift

```diff
@@ -151,4 +135,8 @@ class ArtistViewModel: ObservableObject {
     return setlist.sets?.setsSet?.isEmpty == true
   }
   
+  func isKorean() -> Bool {
+    guard let languageCode = Locale.current.language.languageCode?.identifier else { return false }
+      return languageCode == "ko"
+  }
```

Comment:
앱이 예쁘길래 PR 구경하다가 잠시 눈에 들어와서... 주접 미리 죄송합니다.

```suggestion
  var isKorean: Bool {
    Locale.current.language.languageCode?.identifier == "ko"
  }
```

1. 어떤 액션을 수행하는 것이 아닌 단순한 값을 리턴하는 로직이면 computed property 로 하면 어떨까요? `isEmpty` 처럼요. 
2. 139~140 줄은 결과적으로 `guard let` 을 쓰지 않고 바로 equal 체크하면 어떨까요? `nil` 이면 어차피 `nil = "ko"` 라 `false` 로 리턴될테니